src/main/java/it/gov/pagopa/arc/utils/Utilities.java

```diff
@@ -1,50 +1,14 @@
 package it.gov.pagopa.arc.utils;
 
 
-import it.gov.pagopa.arc.exception.custom.BizEventsInvalidAmountException;
-import it.gov.pagopa.arc.exception.custom.BizEventsInvalidDateException;
 import it.gov.pagopa.arc.exception.custom.ZendeskAssistanceInvalidUserEmailException;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
 
-import java.text.NumberFormat;
-import java.text.ParseException;
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeParseException;
-import java.util.Locale;
-
 @Slf4j
 public class Utilities {
     private Utilities(){}
 
-    /** To convert euro into cents */
-    public static Long euroToCents(String euroString){
```

Comment:
the utilities functions, specially if really generics, could be let here if there is a test on them

Reply:
these utilities have been moved to the Mapper Utilities class